data/json/items/comestibles/bread.json

```diff
@@ -296,12 +296,25 @@
     "calories": 74,
     "description": "A sweet bread, like cake.",
     "price": "272 cent",
-    "price_postapoc": "4 USD",
+    "price_postapoc": "2 USD",
     "material": [ "wheat" ],
-    "volume": "48 ml",
+    "volume": "480 ml",
     "charges": 10,
     "flags": [ "EATEN_HOT", "EATEN_COLD" ],
     "fun": 3,
     "vitamins": [ [ "calcium", 2 ], [ "iron", 6 ] ]
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "bread_canned",
+    "name": { "str_sp": "canned brown bread" },
+    "copy-from": "brown_bread",
+    "spoils_in": "360 days",
```

Comment:
I changed this because I realized that canned bread has 0 preservatives and due to that some bread can spoil faster than others IRL due to the sealed bacteria in it, but since that's not a reality in this game, I just bumped down the expiration date